archlive/airootfs/usr/bin/gpd-setup

```diff
@@ -32,9 +32,7 @@ add_repo() {
 
 set_settings() {
         mkdir -p ${volume}/etc/default && cp /etc/default/tlp ${volume}/etc/default/tlp && \
-        mkdir -p ${volume}/etc/default && cp /etc/default/gpd-fan ${volume}/etc/default/gpd-fan && \
         echo "Successfully defined default settings" || \
-        echo "Unable to defined default settings for tlp and gpd-fan. Please see archwiki page on GPD Pocket."
```

Comment:
Removing this line breaks the try catch that displays the error message. Its better to just remove `and gpd-fan`